packages/tsmuxergui/build.sh

```diff
@@ -10,3 +11,6 @@ TERMUX_PKG_DEPENDS="libc++, qt5-qtbase, qt5-qtdeclarative, qt5-qtmultimedia, qt5
 TERMUX_PKG_BUILD_DEPENDS="freetype, qt5-qtbase-cross-tools, qt5-qttools-cross-tools, zlib"
 TERMUX_PKG_RM_AFTER_INSTALL="bin/tsmuxer"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="-DCMAKE_INSTALL_DATADIR=share -DTSMUXER_GUI=ON"
+termux_step_post_make_install() {
+	mv ${PREFIX}/bin/tsMuxerGUI ${PREFIX}/bin/tsmuxergui
```

Comment:
Please use ${TERMUX_PREFIX} instead of ${PREFIX}, we use $TERMUX_PREFIX in our build scripts.